kivy/uix/modalview.py

```diff
@@ -112,7 +119,8 @@ class ModalView(AnchorLayout):
     defaults to True.
     '''
 
-    attach_to = ObjectProperty(None)
+    # noinspection PyArgumentEqualDefault
```

Comment:
```suggestion
```

Can you please remove these? They shouldn't be in the repo. It's not required to pass our tests and this is a pycharm specific false-positive warning because this code is perfectly good python.

If we start accepting IDE specific flags, or even general flags soon they'll be all over the codebase. The point of linting is to help, not to become an additional piece of code that needs to be managed. I understand it's annoying to see these pycharm highlighting warnings, but they are meant to help and if the goal then becomes to make pycharm show no warnings then it kinda lost the point of being there to help write better code, not wrangle with the IDE.

I'll link to a video that @tshirtman loves to quote about this topic: https://www.youtube.com/watch?v=wf-BqAjZb8M.